ranger/config/rc.conf

```diff
@@ -163,6 +163,15 @@ set draw_progress_bar_in_status_bar true
 # Both combines the two.
 set draw_borders none
 
+# The date format used in the GUI
+set date_format %x
+
+# The time format used in the GUI
+set time_format %X
+
+# The combined date and time format used in the GUI
+set date_plus_time_format %x %X
```

Comment:
`#set date_and_time_format %x %X`
Comment this out and implement defaulting to `date_format time_format` in the code. I don't think we can easily refer to other settings when setting something.